src/engine/BMInterfaceGame.php

```diff
@@ -49,23 +48,43 @@ public function create_game(
         }
 
         try {
-            $gameId = $this->insert_new_game($playerIdArray, $maxWins, $description, $previousGameId);
+            if (!isset($currentPlayerId)) {
+                if ($this->isTest) {
```

Comment:
Or create a different function that is used in the test code to replace the calls to create_game()?